test/mailers/previews/user_mailer_preview.rb

```diff
@@ -40,6 +40,21 @@ def claim_notification_registered
     UserMailer.claim_notification(creator_id, [work.id], true)
   end
 
+  def delete_work
+    work = create(:work)
+    UserMailer.delete_work_notification(work.pseuds.first.user, work)
+  end
+
+  def delete_work_by_co_creator
+    first_creator = create(:user, :for_mailer_preview)
+    second_creator = create(:user, :for_mailer_preview)
+
+    work = create(:work, authors: [first_creator.default_pseud, second_creator.default_pseud])
+
+    User.current_user = second_creator
```

Comment:
This is a possible way to do it, but it feels a bit hacky. What do you think about instead pulling the self vs other check out of the mailer erb files into work.rb and only passing the result into the mailer method?